spec/magma_child_attribute_spec.rb

```diff
@@ -12,8 +12,8 @@
         .and_return true
     end
 
-    it 'calls update or create on the link_model' do
-      child_attribute.update_record(record, link)
+    xit 'calls update or create on the link_model' do
```

Comment:
Is this test supposed to be skipped?

Reply:
I should just remove this test -- the method doesn't exist any more on the attributes, and link updates wouldn't be handled by the `update_or_create` method anyways...